docs/content/docs/concepts/overview.md

```diff
@@ -36,7 +36,7 @@ Flink Kubernetes Operator aims to capture the responsibilities of a human operat
   - Stateful and stateless application upgrades
   - Triggering and managing savepoints
   - Handling errors, rolling-back broken upgrades
-- Multiple Flink version support: v1.16, v1.17, v1.18, v1.19
+- Multiple Flink version support: v1.16, v1.17, v1.18, v1.19, v1.20
```

Comment:
To avoid confusion, I would also suggest backporting this commit to the 1.10 release branch (if that's how the 1.10 docs can be updated).

Reply:
Sounds make sense, I have cherry picked it to 1.10.